core/services/chainlink/config_test.go

```diff
@@ -1294,19 +1294,19 @@ func TestConfig_Validate(t *testing.T) {
 		- 4: 2 errors:
 			- ChainID: missing: required for all chains
 			- Nodes: missing: must have at least one node
-	- Cosmos: 5 errors:
+	- Cosmos: 4 errors:
 		- 1.ChainID: invalid value (Malaga-420): duplicate - must be unique
-		- 0.Nodes.1.Name: invalid value (test): duplicate - must be unique
+	0.Nodes.1.Name: invalid value (test): duplicate - must be unique
 		- 0.Nodes: 2 errors:
 				- 0.TendermintURL: missing: required for all nodes
 				- 1.TendermintURL: missing: required for all nodes
 		- 1.Nodes: missing: must have at least one node
 		- 2: 2 errors:
 			- ChainID: missing: required for all chains
 			- Nodes: missing: must have at least one node
-	- Solana: 5 errors:
+	- Solana: 4 errors:
 		- 1.ChainID: invalid value (mainnet): duplicate - must be unique
-		- 1.Nodes.1.Name: invalid value (bar): duplicate - must be unique
```

Comment:
Idk what to do with this, Solana PR that should fix this is not yet merged, same for Cosmos